plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -131,14 +132,21 @@ export const TechDocsReaderPageHeader = (
         <HeaderLabel
           label=""
           value={
-            <a
-              href={locationMetadata.target}
-              target="_blank"
-              rel="noopener noreferrer"
+            <Grid
+              container
+              direction="row"
```

Comment:
Since the other labels are aligned vertically, could we make this one vertical as well by using `direction="column"`?

Reply:
Sure, I could try and let you know the final result